blivet/__init__.py

```diff
@@ -202,6 +202,11 @@ def storageInitialize(storage, ksdata, protected):
 
     # kickstart uses all the disks
     if flags.automated_install:
+        # ignore OEMDRV disk
```

Comment:
I think because it resolves reporters / customer's case. Do you think we should apply it to all installs? If so, to me it seems that just moving the patched code before the automated_install if block should do the job then.